apps/portals/b2ai.standards/src/config/resources.ts

```diff
@@ -1,71 +1,93 @@
-  // UseCase: { name: 'UseCase', id: 'syn63096837' }, // not using this, maybe will in the future?
+  // Organization: { name: 'Organization', id: 'syn63096836.31' },
+  Organization_denormalized: {
+    name: 'Organization',
+    id: 'syn69693360.20', // current version of Organization_denormalized
+  },
+  D4D_content: {
+    name: 'D4D_content',
+    id: 'syn68885644.5', // current version of D4D_content
+  },
+  // UseCase: { name: 'UseCase', id: 'syn63096837' }, // not using this, might in the future?
 }
 
-export const DST_TABLE_COLUMN_NAMES = {
-  ID: 'id',
-  ACRONYM: 'acronym',
-  NAME: 'name',
-  CATEGORY: 'category',
-  DESCRIPTION: 'description',
-  COLLECTIONS: 'collections',
-  HAS_AI_APPLICATION: 'hasAIApplication',
-  MATURE: 'mature',
-  CONCERNS_DATA_TOPIC: 'concerns_data_topic',
-  HAS_RELEVANT_DATA_SUBSTRATE: 'has_relevant_data_substrate',
-  HAS_RELEVANT_ORGANIZATION: 'has_relevant_organization',
-  RELEVANT_ORG_LINKS: 'relevantOrgLinks',
-  RESPONSIBLE_ORG_LINKS: 'responsibleOrgLinks',
-  RELEVANT_ORG_NAMES: 'relevantOrgNames',
-  RESPONSIBLE_ORGANIZATION: 'responsible_organization',
-  IS_OPEN: 'isOpen',
-  REGISTRATION: 'registration',
-  URL: 'URL',
-  FORMAL_SPEC: 'formalSpec',
-  PUBLICATION: 'publication',
-  TRAINING_RESOURCES: 'trainingResources',
-  SUBCLASS_OF: 'subclassOf',
-  CONTRIBUTION_dATE: 'contributionDate',
-  RELATED_TO: 'relatedTo',
-  USED_IN_BRIDGE2AI: 'usedInBridge2AI',
-}
-export const ORG_TABLE_COLUMN_NAMES = {
+export type ColumnConsts = { ID: string } & Record<string, string>
+
+export const ORG_TABLE_COLUMN_CONSTS: ColumnConsts & {
+  NAME: string
+  DESCRIPTION: string
+} = {
   ID: 'id',
   NAME: 'name',
   DESCRIPTION: 'description',
   ROR_ID: 'ror_id',
   WIKIDATA_ID: 'wikidata_id',
   URL: 'url',
   SUBCLASS_OF: 'subclass_of',
-}
-export const ORG_TABLE_JSON_COLUMNS = []
+  // new (and likely to change):
+  MAIN_ORGANIZATION_JSON: 'main_organization_json',
+  ASSOCIATED_ORGANIZATION_JSON: 'associated_organization_json',
+  RELEVANT_STANDARDS: 'relevant_standards',
+  RELEVANT_STANDARDS_JSON: 'relevant_standards_json',
+  GOVERNED_STANDARDS: 'governed_standards',
+  GOVERNED_STANDARDS_JSON: 'governed_standards_json',
+  DATASETS: 'datasets',
+  DATASET_NAMES: 'dataset_names',
+  DATASET_JSON: 'dataset_json',
+  D4D: 'd4d',
+} as const
+tableInfo.Organization_denormalized.columnConsts = ORG_TABLE_COLUMN_CONSTS
 
 export const organizationDetailsPageSQL = `
   SELECT  
-    ${Object.values(ORG_TABLE_COLUMN_NAMES).join(', ')},
+    ${Object.values(ORG_TABLE_COLUMN_CONSTS).join(', ')},
     concat('/Explore/Organization/OrganizationDetailsPage?id=', ${
-      ORG_TABLE_COLUMN_NAMES.ID
+      ORG_TABLE_COLUMN_CONSTS.ID
     }) AS orgPageLink
-  FROM ${TABLE_IDS.Organization.id}`
+  FROM ${tableInfo.Organization_denormalized.id}`
+tableInfo.Organization_denormalized.queries ??= {}
+tableInfo.Organization_denormalized.queries.detailsSQL =
+  organizationDetailsPageSQL
```

Comment:
Kind of odd to dynamically set these, but I understand why you had to do it. Seems like this tableInfo needs to be 'normalized' itself